themes/edx.org/lms/templates/dashboard.html

```diff
@@ -180,6 +180,19 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
       </ul>
     </section>
   </section>
+  % if xseries_credentials:
```

Comment:
@mattdrayer if you are talking about some kind of configuration to enable displaying of xseries data then we already have configuration models in LMS for that.
For example if a user hasn't enabled the `Programs api config` and `Credentials api config` then this section won't be displayed at all.